joomla/joomla-cms
# [com_fields] Custom Fields Media Field if class is empty still class is shown as attribute

Pull Request for Issue #13400 

### Summary of Changes

Don't let the class tag alone without content

### Testing Instructions

Create an media field and don´t fill the image class field.

There is an empty class attribute shown in the Source code
![image](https://cloud.githubusercontent.com/assets/828371/21544078/641d240a-cdcc-11e6-93ec-d3e7974cf5e2.png)

Apply the patch and make sure there is no class tag anymore but if you set a class it comes back :smile: 

### Documentation Changes Required

none
